app/Console/Commands/EnrichmentCommand.php

```diff
@@ -8,24 +8,41 @@
 
 class EnrichmentCommand extends Command
 {
-    protected $signature = 'osmfeatures:enrich';
+    protected $signature = 'osmfeatures:enrich {osmid?*}';
 
-    protected $description = 'Enrich osmfeatures places with data from Wikipedia, Wikidata, and Wikimedia Commons';
+    protected $description = 'Enrich osmfeatures places with data from Wikipedia, Wikidata, and Wikimedia Commons. Optionally provide a list of OSM IDs separated by space.';
 
     public function handle()
     {
+        $osmids = $this->argument('osmid');
         $jobCount = 0;
 
-        Place::chunk(100, function ($places) use (&$jobCount) {
-            foreach ($places as $place) {
-                EnrichmentJob::dispatch($place);
-                $jobCount++;
+        if (!empty($osmids)) {
```

Comment:
definire $places con tutti i places e gestire tramite if la ridefinizione di $places solo per determinate osmid, in maniera tale di eliminare la ridondanza di codice